stylesheets/utils/utilities.css

```diff
@@ -114,3 +122,16 @@ h6 {
   font-size: 1rem;
   font-weight: 500;
 }
+
+/* align-children */
+
+.children-center{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.children-stacked {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
You can create utility classes for all of these.

Reply:
these itself are util classes, please refer the file name

Reply:
Got it.